docs/production-deployment/cloud/tcld/user-group.mdx

```diff
@@ -0,0 +1,151 @@
+---
+id: user-group
+title: tcld user group command reference
+sidebar_label: user-group
+slug: /cloud/tcld/user-group/
+toc_max_heading_level: 4
+keywords:
+  - cli reference
+  - tcld
+tags:
+  - Temporal Cloud
+  - tcld
+  - User groups
+---
+
+The `tcld user-group` commands manage user groups in Temporal Cloud.
+
+Alias: `ug`
+
+- [tcld user-group add-users](#add-users)
+- [tcld user-group create](#create)
+- [tcld user-group get](#get)
+- [tcld user-group list](#list)
+- [tcld user-group list-members](#list-members)
+- [tcld user-group remove-users](#remove-users)
+- [tcld user-group set-access](#set-access)
+
+## add-users
+
+The `tcld user-group add-users` command adds users to the specified user group in Temporal Cloud.
+You must set `--group-id` to specify the group to add users to.
+
+Alias: `au`
+
+The following flags control the behavior of the command.
+
+#### --group-id (-id)
+
+Specify the ID of the group to add users to.
+
+#### --user-email (-e)
+
+Specify the email of the user to add. This flag can be specified multiple times to add
+multiple users in one command
+
+
+## create
+
+Creates a user group.
+
+Alias: `c`
+
+The following flags control the behavior of the command.
+
+#### --display-name
+
+The display name of the group.
+
+#### --account-role
+
+The account role that the group should have. One of `admin`, `read`, `developer`, `owner`, `financeadmin`, `none`.
+
+#### --namespace-role (-nr)
```

Comment:
Do we call namespace level access `role` or `permissions`? Seems like we have both in this PR